assets/helpers/abTests/helpers.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+
+import type { Participations } from './abtest';
+
+function getVariantForTest(participations: Participations, test: string) {
+  return participations ? participations[test] : null;
+}
+
+function getPaymentLogosTestVariant(participations: Participations) {
+  return getVariantForTest(participations, 'paymentLogosTest');
+}
+
+function inPaymentLogosTest(participations: Participations) {
+  return getPaymentLogosTestVariant(participations) === 'variant';
+}
+
+export {
+  getVariantForTest,
```

Comment:
Are we using this function elsewhere? I am not able to find it, if we are not using this function I would delete it from the export object.

Reply:
It is also used here: https://github.com/guardian/support-frontend/blob/74ccf0d406f564500e63c692136bd8bdbcf75022/assets/helpers/abTests/helpers.js#L14

I think it's worth keeping because otherwise you have to hardcode the test name string in 3 places.